github-actions/scan/README.adoc

```diff
@@ -81,6 +81,16 @@ To be able to use this action you need a SecHub project. Check the https://merce
     fail-job-with-findings: true
 ----
 
+*Note:* The following variables take priority over the configuration file:
+
+- `USERID`
```

Comment:
The user cannot set this variable directly - so this note will not help.
You should mention above parameters. In this case:
```suggestion
- `user:`
```